### PR TITLE
Styles MHV Home links to be 20px (1.25rem)

### PR DESCRIPTION
**Pull request by radavis**
## Summary

**Styles MHV Home links to be 20px (1.25rem)**

- Changes font size of tool links and spotlight links from 17px to 20px
- Tweaks the card margins so that the arrow/chevron isn't orphaned
- Other stylistic tweaks

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#85688

## Testing done

- localhost

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |     va-top-mobile-1--my-health   |   va-top-mobile-1--my-health--registered     |
| Desktop |  va-top-desktop-2--my-health      |  va-top-desktop-2--my-health--registered     |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

File: src/applications/mhv-landing-page/components/HubLinks.jsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ const HubSection = ({ title, links }) => {
   const listItems = links.map(({ href, text }, index) => (
     <li key={`${href}--${index}`}>
       <va-link
-        class="mhv-c-link"
```
[the remark this answers is not in the document]
Comment (radavis) [reply src/applications/mhv-landing-page/components/HubLinks.jsx:9]: Swapped the `<va-link />` web component for an anchor tag.

**Review by fmccaf1 — COMMENTED** [review]
I'm not clear if I should be seeing something different now that you swapped the anchor tag back in.... but looking at the links in the 6 cards, that looks good! Let me know if there is anything else I should be reviewing besides those card links

**Review by carlosfelix2 — CHANGES_REQUESTED** [review]
Looks like the subscribe to the newsletter portion is still 16px. Change to 20px

**Comment by radavis** [comment]
@carlosfelix2 I didn't see that ask called out in the ACs. https://github.com/department-of-veterans-affairs/va.gov-team/issues/85688

File: src/applications/mhv-landing-page/components/HubLinks.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (radavis) [reply src/applications/mhv-landing-page/components/HubLinks.jsx:9]: And then swapped it back :upside_down_face:

**Review by carlosfelix2 — APPROVED** [review]
Looks good. Some fonts are 16.96px, but unsure if that is something you can control. Sent message in Slack about it.

**Comment by fmccaf1** [comment]
This looks good! Looks like all the changes are in and everything is looking correctly sized.

**Pull request merged**